interp/src/debugger/cidr.rs

```diff
@@ -1,5 +1,3 @@
-use std::{cell::Ref, collections::HashMap, rc::Rc};
-
```

Comment:
In the future, try to avoid rearranging things like the import lists unless there's a compelling reason to do so, since it creates extra noise when reviewing your PR